### PR TITLE
Replace TreeView in Wizard with HTML/CSS.

### PR DESCRIPTION
**Pull request by dlabrecq**
PTNFLY-1620

### CODE REVIEW

**Comment by dlabrecq** [comment]
Click on the "Launch wizard showing review step" button.

https://rawgit.com/dlabrecq/patternfly/PTNFLY-1620/dist/tests/wizard.html

**Comment by dlabrecq** [comment]
@jeff-phillips-18 @LHinson please review.

screen shot 2016-07-22 at 4 46 06 pm

**Comment by LHinson** [comment]
@kybaker will you double check this from a visual perspective since SJ is out? I'm not sure because the mockups in the visual jira (https://patternfly.atlassian.net/browse/PTNFLY-1241) do not show this many level deep. 

Otherwise, the rest looks good to me.

**Comment by srambach** [comment]
LGTM. Nicely done.

File: dist/css/patternfly-additions.css [hunk 1]
```diff
@@ -5926,21 +5926,103 @@ a.disabled {
...
+  border: none;
+  margin-bottom: 0;
+}
+.wizard-pf-review-steps > ul > li {
+  position: relative;
+  float: left;
+  margin: 0;
+  padding-top: 0;
+  line-height: 15px;
+  width: 100%;
```
Comment (jeff-phillips-18) [inline dist/css/patternfly-additions.css:5943]: Please alphabetize

File: dist/css/patternfly-additions.css [hunk 2]
```diff
@@ -5926,21 +5926,103 @@ a.disabled {
...
-  font-size: 16px;
-  margin-left: 20px;
-  padding: .5em .5em;
   text-decoration: none;
+  font-size: 16px;
+  transition: 0.25s;
+  cursor: pointer;
+  padding-left: 30px;
+  padding-right: 5px;
+}
```
Comment (jeff-phillips-18) [inline dist/css/patternfly-additions.css:5953]: Alphabetize

File: less/wizard.less [hunk 3]
```diff
@@ -205,22 +205,123 @@
...
-    font-size: 16px;
-    margin-left: 20px;
-    padding: .5em .5em;
-    text-decoration: none;
-    a {
-      color: @color-pf-black;
-      text-decoration: none;
+.wizard-pf-review-steps {
+  list-style: none;
+
```
Comment (jeff-phillips-18) [inline less/wizard.less]: Please remove blank lines

**Comment by LHinson** [comment]
After syncing with Dan, I'm now aware that this was demoed and approved by @kybaker from a visual standpoint. I thing this wraps it up. LGTM!

**Comment by dlabrecq** [comment]
@jeff-phillips-18 I alphabetized and removed white spaces.

**Comment by jeff-phillips-18** [comment]
LGTM

**Pull request merged**